daux/docs/1100_Distributed_Query/100_How_Distributed_Query_Works/200_Querying_with_Criteria_API.md

```diff
@@ -46,8 +46,8 @@ characters,  (underscore) is placeholder for only one character.
 
 ***RELATED INFORMATION*** 
 
-*Please see the <a href="https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/query/Predicates.java" target="_blank">
-Predicates class</a> for all predicates provided.*
+*Please see the <a href="http://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/query/Predicates.html">
```

Comment:
Is there any reason why not to use a MarkDown link instead of HTML?

Reply:
To open the page in new window.